baseline-error-prone/build.gradle

```diff
@@ -36,14 +36,30 @@ dependencies {
     compileOnly 'com.google.auto.service:auto-service'
 }
 
-// Incorrectly identifies tests as junit4 usage
-tasks.checkJUnitDependencies.enabled = false
-tasks.withType(JavaCompile) {
-    options.errorprone {
+tasks.withType(JavaCompile).configureEach {
+    sourceCompatibility = 11
+    options.compilerArgs += [
+        '--add-exports=jdk.compiler/com.sun.tools.javac.api=ALL-UNNAMED',
+        '--add-exports=jdk.compiler/com.sun.tools.javac.file=ALL-UNNAMED',
+        '--add-exports=jdk.compiler/com.sun.tools.javac.code=ALL-UNNAMED',
+        '--add-exports=jdk.compiler/com.sun.tools.javac.util=ALL-UNNAMED',
+        '--add-exports=jdk.compiler/com.sun.tools.javac.comp=ALL-UNNAMED',
+        '--add-exports=jdk.compiler/com.sun.tools.javac.main=ALL-UNNAMED',
+        '--add-exports=jdk.compiler/com.sun.tools.javac.model=ALL-UNNAMED',
+        '--add-exports=jdk.compiler/com.sun.tools.javac.parser=ALL-UNNAMED',
+        '--add-exports=jdk.compiler/com.sun.tools.javac.processing=ALL-UNNAMED',
+        '--add-exports=jdk.compiler/com.sun.tools.javac.tree=ALL-UNNAMED',
+        '--add-exports=jdk.compiler/com.sun.tools.javac.util=ALL-UNNAMED'
+    ]
+   options.errorprone {
         check("StrictUnusedVariable", CheckSeverity.OFF)
     }
 }
 
+// Incorrectly identifies tests as junit4 usage
+tasks.checkJUnitDependencies.enabled = false
+tasks.javadoc.enabled = false
```

Comment:
I have no idea how to pass `--add-exports` flags to javadoc in gradle (options are definitely not being respected) and at this point I have given up and disabled javadoc for errorprone.